pkg/logp/logp.go

```diff
@@ -0,0 +1,21 @@
+package logp
+
+import "github.com/sirupsen/logrus"
```

Comment:
logrus is in maintenance mode. Gateway operator is using zap instead. Does this package needs it's own logger or can we just provide it as a dependency?

Reply:
We can use whatever. I don't recall us reaching the consensus around logging packages that we use.

What's your suggestion?

Reply:
There was not. Was deprioritized.

Reply:
Renamed to `log` and created #48 to track changing `logrus` to a different package.